task5/src/main/test/com/clouway/adapter/http/BankOperationHandlerTest.java

```diff
@@ -0,0 +1,200 @@
+package com.clouway.adapter.http;
+
+import com.clouway.core.CookieFinder;
+import com.clouway.core.FundsRepository;
+import com.clouway.core.Session;
+import com.clouway.core.SessionRepository;
+import com.clouway.core.UserRepository;
+import com.clouway.core.Validator;
+import org.jmock.Expectations;
+import org.jmock.auto.Mock;
+import org.jmock.integration.junit4.JUnitRuleMockery;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+import javax.servlet.http.Cookie;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Created by Kristiyan Petkov  <kristiqn.l.petkov@gmail.com> on 08.06.16.
+ */
+public class BankOperationHandlerTest {
+  private BankOperationHandler bankOperationHandler;
+  private String amount;
+  private String email;
+  private Double amountInDouble;
+  private Cookie[] cookies;
+  private Cookie cookie;
+
+
+  @Rule
+  public JUnitRuleMockery context = new JUnitRuleMockery();
+
+  @Mock
+  HttpServletRequest request;
+
+  @Mock
+  HttpServletResponse response;
+
+  @Mock
+  Validator validator;
+
+  @Mock
+  FundsRepository fundsRepository;
+
+  @Mock
+  CookieFinder cookieFinder;
+
+  @Mock
+  SessionRepository sessionRepository;
+
+  @Before
+  public void setUp() {
+    bankOperationHandler = new BankOperationHandler(fundsRepository, validator, cookieFinder, sessionRepository);
+    amount = "200.00";
+    email = "admin@abv.bg";
+    amountInDouble = 200.00D;
+    cookies = new Cookie[]{};
+    cookie = new Cookie("sessionId", "3131-3344-6667-8843");
+  }
+
+  @Test
+  public void makeDepositHappyPath() throws Exception {
```

Comment:
depositMoney - is enought. we already discussed shorter names are better. suggestion